config/proto/targets.proto

```diff
@@ -0,0 +1,44 @@
+/*
+  This proto defines the config for a Target. 
+  Targets are used by Hermes and Cloudprober to identify storage systems to monitor.
+*/
+syntax = "proto3";
+
+package hermes;
+
+option go_package = "github.com/googleinterns/step224-2020/targets/proto";
+
+// Definition of a Target, a storage system to be probed by a specific probe.
```

Comment:
This sounds a bit confusing. Maybe something along the lines of: Target contains all the metadata necessary for Hermes to establish a connection to a storage system. Every probe request will require a (at least one) target.

Reply:
```suggestion
// Target contains all of the metadata necessary for Hermes to establish a connection to a storage system.
// Every probe request will require one or more targets.
```